vspace/src/main/webapp/WEB-INF/views/staff/modules/slides/slide.jsp

```diff
@@ -44,6 +44,97 @@ function uploadImage() {
 } 
 	
 $(document).ready(function() {
+	//-------- edit description --------
+	$("#submitDescription").hide()
+	$("#editDescription").click(function() {
+		var val = $("#description").val();
+		$("#description").html(function (i, val) {
+	        return '<textarea id="newDescription" style="width:250%; height:200%;" class="form-control" type="text">'+val+'</textarea>';
+	    });
+		$("#editDescription").hide()
+		$("#submitDescription").show()
+		
+	});
+	
+	$("#submitDescription").click(function() {
+		var formData = new FormData();
+		formData.append('description', $("#newDescription").val());
+		$.ajax({
+		url: "<c:url value="/staff/module/${module.id}/slide/${slide.id}/edit/description?${_csrf.parameterName}=${_csrf.token}" />",
+		type: 'POST',
+		cache       : false,
+		contentType : false,
+		processData : false,
+		data: formData,
+		enctype: 'multipart/form-data',
+		success: function(data) {
+			// replace text box with new description
+			$("#submitDescription").hide()
+			$("#editDescription").show()
+			var val = $("#newDescription").val();
+			$("#newDescription").remove()
+			$("#description").text(val)
+			
+		},
+		error: function(data) {
+			console.log(data);
+			var alert = $('<div class="alert alert-danger alert-dismissible fade show" role="alert"><p>We are sorry but something went wrong. Please try again later.</p><button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">&times;</span></button></div>');
+			$('.error').append(alert); 
+			$(".error").delay(4000).slideUp(500, function(){
+			    $(".error").empty();
+			});
+		}
+	});
+		
+  	});
+	
+	//------- edit title --------
+	$("#submitTitle").hide()
+	$("#editTitle").click(function() {
+		$('<div class="col-4"><input id="newTitle" class="form-control" type="text"></div>').insertAfter( "#title" );
+		var val = $("#title").text()
+		var val2 = val.split(": ")[1] 
```

Comment:
why do you need to split this? also, bad variable naming.

Reply:
The title is a header tag so the text value returns 'Slide: Test 3', I split it on the colon in order to just get Test 3 to populate the textbox with. Otherwise, I would populate the text box with Slide: Test 3. I meant to change the names, I ended combining that into one line.